akka-http-core/src/main/scala/akka/http/impl/engine/http2/ResponseRendering.scala

```diff
@@ -71,28 +93,37 @@ private[http2] object ResponseRendering {
     headerPairs.result()
   }
 
+  /**
+   * Mutates `headerPairs` adding any valid header from `headersSeq`.
+   * @param peerIdHeader a header providing extra information (e.g. vendor and version) about the
+   *                     peer. For example, a User-Agent on the client or a Server header on the server.
+   */
   private[http2] def renderHeaders(
     headersSeq:   immutable.Seq[HttpHeader],
     headerPairs:  VectorBuilder[(String, String)],
-    serverHeader: Option[(String, String)],
+    peerIdHeader: Option[(String, String)],
```

Comment:
renamed from `server` to `peerId` since that may be used for `Server` or `User-Agent` now.